sdk/storage/AzureStorageBlob/README.md

```diff
@@ -61,13 +61,22 @@ az storage account create -n my-storage-account-name -g my-resource-group
 ### Create the client
 The Azure Storage Blobs client library for iOS allows you to interact with blob storage containers and blobs.
 Interaction with these resources starts with an instance of a [client](#client). To create a client object, you will
-need the storage account's blob service account URL and a credential that allows you to access the storage account:
+need a credential that allows you to access the storage account:
+
+```swift
+import AzureStorageBlob
+
+let credential = ...
+let client = StorageBlobClient(credential: credential)
```

Comment:
Restoration ID is required for this initializer. Also, we should make clear that the signature without either baseUrl or accountUrl only works with `StorageSASCredential`.